gprofiler/metadata/system_metadata.py

```diff
@@ -64,7 +64,7 @@ def get_run_mode() -> str:
         return "k8s"
     elif os.getenv("GPROFILER_IN_CONTAINER") is not None:  # set by our Dockerfile
         return "container"
-    elif os.getenv("STATICX_BUNDLE_DIR") is not None:  # set by staticx
+    elif is_pyinstaller():
```

Comment:
PyInstaller is always used right? There's no standalone build without it?

Reply:
Yup. And staticx is only x86_64 now